Rovers/Rovers.tex

```diff
@@ -0,0 +1,405 @@
+We will be using time as a metric as well as binary metrics for completion of the parts of the competition. 
+Some features might behave differently than expected or planned these features will be evaluated accordingly.
+This includes ’does the rocker-bogie system works as intended’, ’will the rover stop when directed at an obstacle’, etc. 
+The details are defined in the minimum viable product section.
+
+\subsection{Minimum Viable Product}
+\label{subsec:mvp}
+The minimum viable product (MVP) has been defined by the basics of what can be called a 'functioning rover'. This includes features like suspension, motors, power and controls. The MVP is defined by the following features: 
+
+\begin{table}[hb!]
+    \caption{Minimum Viable Product}
+    \centering
+    {\renewcommand{\arraystretch}{1.5}
+    \begin{tabularx}{\linewidth}{LL} 
+    \hline
+    \textbf{Feature} & \textbf{Reason} \\
+    \hline
+    Rocker-bogie-like suspension & This is the NASA preferred rover suspension style. \\
+    Ability to control remotely (such as a gamepad) & Remote control is essential for rovers. \\
+    Ability to traverse terrain such as gravel, cement, and asphalt & Multi-terrain traversal is important for rovers. \\
+    Technical report on GitHub & This will allow for future teams to easily learn from this teams experience. \\
+    Rover hardware & Having a physical rover is important for a rover project as it allows for lessons to be learned in construction. \\ 
+    Present at Imagine RIT & This is essential for promotional value. \\
+    Partial self-driving & Partially autonomous navigation for future URC readiness. \\
+    \hline
+    \end{tabularx}
+    }
+\label{tab:mvp-one}
+\end{table}
+
+\subsection{Improvements}
+\label{subsec:improvements}
+The MVP is intended to be flexible in implementation depending on funding. 
+The purpose of the project is to gain experience in rovers and robotics and this can be done in a number of form factors.
+The size and materials are not defined because of this. 
+Should the funding allow for it there are many improvements that can be made beyond the MVP. 
+This allows for more experience to be gained, thus fulfilling the goals of the project.
+
+\begin{table}[ht!]
+    \caption{Improvements}
+    \centering
+    {\renewcommand{\arraystretch}{1.5}
+    \begin{tabularx}{\linewidth}{LL}
+    \hline
+    \textbf{Feature} & \textbf{Reason} \\  
+    \hline
+    Lidar based CV & Autonomous navigation for future URC readiness. \\
+    Improved navigation with use of GPS system & Additional navigation options. \\
+    Robotic arm with 3+ degrees of freedom & URC readiness and rover capabilities. \\ 
+    Drill or grasper attachment & Increased capabilities of the robotic arm. \\
+    Soil sample analysis with Archimedes' screw & Scientific possibilities. \\
+    Larger and or faster rover & Able to traverse obstacles of greater magnitude or hold more scientific equipment for future expansion. \\
```

Comment:
Size and weight should be dependent variables on capability. Building something larger for the sake of making it larger is relatively easy. Building a rover that is just big enough to support its required functionality is a challenge.

Reply:
The larger rover also allows space for more enhancements.